src/main/java/mate/jdbc/controller/RegistrationController.java

```diff
@@ -40,10 +41,20 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String password = req.getParameter("password");
         String confirmPassword = req.getParameter("confirm");
         String encrypt = HashUtils.encrypt(password);
+        String user = Role.USER.toString();
+        String admin = Role.ADMIN.toString();
```

Comment:
@EliosPride take role from param "button" and parse to enum

Reply:
@Garret-vin +